go/pkg/cas/upload.go

```diff
@@ -543,7 +543,7 @@ func (u *uploader) visitPath(ctx context.Context, absPath string, info os.FileIn
 			return nil, err
 		}
 	}
-
+	
```

Comment:
This is probably breaking CI. `gofmt -w .` should fix it.

Reply:
Done.